assets/src/pages/room/components/StreamPlayer/UnpinnedTilesSection.tsx

```diff
@@ -0,0 +1,119 @@
+import React, { FC, useMemo } from "react";
+import MediaPlayerTile from "./MediaPlayerTile";
+import { MembraneWebRTC, TrackEncoding } from "@jellyfish-dev/membrane-webrtc-js";
+import clsx from "clsx";
+import { MediaPlayerTileConfig, TrackWithId } from "../../../types";
+import PeerInfoLayer from "./PeerInfoLayer";
+import { GridConfigType, getGridConfig } from "../../../../features/room-page/utils/getVideoGridConfig";
+import NameTag from "../../../../features/room-page/components/NameTag";
+import InitialsImage from "../../../../features/room-page/components/InitialsImage";
+import { PinTileLayer } from "../../../../features/room-page/components/PinComponents";
+import {
+  DisabledMicIcon,
+  isLoading,
+  showDisabledIcon,
+} from "../../../../features/room-page/components/DisabledTrackIcon";
+
+const getGridStyle = (
```

Comment:
It looks like a helper function so, please move it to `features/room-page/utils`